docs/policy_authoring.md

```diff
@@ -81,7 +82,20 @@ to use standard OPA tooling.
 You can generate a fixture using the `fixture` command.  For example, we can
 generate a fixture for the example terraform file we are using like this:
 
-    ./policy-engine fixture examples/main.tf >examples/tests/fixture.json
+    ./policy-engine fixture examples/main.tf >examples/tests/fixture.rego
+
+You can also fetch resources from the Snyk Cloud API and use them to generate
```

Comment:
I would add a new subheader for this so it's easy to link to these instructions.

Reply:
Thanks! Fixed in 8dba15a.